packages/filebrowser-extension/src/index.ts

```diff
@@ -65,6 +65,7 @@ import {
 import { find, map } from '@lumino/algorithm';
 import { CommandRegistry } from '@lumino/commands';
 import { ContextMenu } from '@lumino/widgets';
+import { posix } from 'path';
```

Comment:
This will not work as the code is running in the browser, not in a Node runtime.

Reply:
Function `PathExt.join` was wrapped by it, and it can be execed in browser.

Reply:
No, it cannot be used in browser because browsers do not implement Node runtime. You would need to add a dependency on `path-browserify`, which is what the `@jupyterlab/coreutils` package which implements `PathExt.join` does:

https://github.com/jupyterlab/jupyterlab/blob/3c3c65ae03a758c620d8dd19678cb891d6bf0f20/packages/coreutils/package.json#L47

However, ideally this implementation detail and dependency would not be propagated to filebrowser-extension, and either a new argument would be added to one of `PathExt` methods, or a new method created altogether and used in here.